app/matplotlib.ipynb

```diff
@@ -339,18 +355,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
-   "outputs": [],
    "source": [
-    "# write your code here\n"
+    "df = pd.read_csv(\"../data/customer_data.csv\")\n",
```

Comment:
Ensure that the file path '../data/customer_data.csv' is correct and that the file exists in the specified location. Otherwise, this line will cause an error when trying to read the CSV file.